### DISCUSSION TITLE
Tailwind licence and usage of TailGrid

### DISCUSSION THREAD

**Discussion by 0ximDigital** [tailwindlabs/tailwindcss#13221] [Help]
Hi Everyone!

Tailwind license explicitly states under the _Examples of usage not allowed by the license:_ section the following:

- Creating a UI library using Tailwind UI components and making it available either for sale or for free.

We also have the [TailGrids product](https://tailgrids.com/) that is doing exactly that - It created UI components based on Tailwind CSS and is [selling them](https://tailgrids.com/pricing).
Is that allowed?

Could we base our product on the TailGrid components?

Many Thanks

**Comment by wongjn** [comment]
You'd probably need to contact the developers of TailGrids for clarification.
